tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -37,4 +37,43 @@ Check Image Build Status
   Run Keyword If    "${build_status}" == "${target_status}"
   ...    Log   The '${build_name}' image match the expected target status
   ...  ELSE
-  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
\ No newline at end of file
+  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
+
+Search Last Build
+    [Documentation]    Returns latest(sorted by creation time) build which match the ${build-name-includes}
+    [Arguments]    ${namespace}    ${build-name-includes}
+    ${build} =    Run    oc get builds --sort-by=.metadata.creationTimestamp -n ${namespace} | grep ${build-name-includes} | awk '{print $1;}'
+    @{builds} =  Split String  ${build}  \n
+    [Return]    ${builds}[-1]
+
+Deletes Build
+    [Documentation]    Deletes the build ${build_name} in ${namespace}
+    [Arguments]    ${namespace}    ${build_name}
+    OpenShiftCLI.Delete    kind=Build    namespace=${namespace}    field_selector=metadata.name==${build_name}
+
+Starts New Build
```

Comment:
The keyword name should be `Start New Build` (Start without s). In the documentation section Starts (with s) is correct